ietf/doc/views_ballot.py

```diff
@@ -403,7 +403,7 @@ def clear_ballot(request, name, ballot_type_slug):
         by = request.user.person
         if close_ballot(doc, by, ballot_type_slug):
             create_ballot_if_not_open(request, doc, by, ballot_type_slug)
-        if doc.get_state('draft-iesg').slug == 'defer':
+        if doc.get_state("statchg").slug == "defer":
```

Comment:
This shoul have a guard against `doc.get_state()` returning `None`. (I think this is the cause of the failing test)

Reply:
hmm - that just exposes another subtle bug - the check should be against the ballot_type_slug, and if there is no state, the view had best do nothing (404 would be right I think).